jhu-sheridan-libraries/stfrancis-lutece-site
# Reorganize repo

Make lutece-site the root of the repo by splitting off lutec-init as https://github.com/jhu-sheridan-libraries/lutece-init. Also some unused files are removed and documentation updated.

In order to test:
1. Build the war
1. cp target/*.war lutece.war
1. Run the war in docker
1. Verify site is running

Closes #31
